### ISSUE TITLE
Integrate with yjs for replication?

### ISSUE THREAD

**Issue by KyleAMathews** [tinyplex/tinybase#52]
Curious if someone's looked into this. Would complement tinybase really well!

**Comment by jamesgpearce** [comment]
Yes... I've been riffing on my own CRDT implementation in https://github.com/tinyplex/tinysync but jury is still out: it might be better to figure out a YJS connector instead.

**Comment by KyleAMathews** [comment]
🔥 great to know!

**Comment by jamesgpearce** [comment]
OK, please take a look at the new 4.0 beta and kick the tires! https://github.com/tinyplex/tinybase/releases/tag/v4.0.0-beta.2
